File: 3870-Count-Commas-in-Range/main.py
class Solution:
    def countCommas(self, n: int) -> int:
        if n < 1000:
            return 0
        return n - 999
        
if __name__ == '__main__':
    solution = Solution()
    result = solution.countCommas(n=1002)
    print(result)

File: 3870-Count-Commas-in-Range/main.java
class Solution {
    public int countCommas(int n) {
        return n < 1000 ? 0 : n - 999;
    }
}

class Test {
    public static void main(String[] args) {
        Solution solution = new Solution();
        int result = solution.countCommas(1002);
        System.out.println(result);
    }
}